unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/Scripts/GodMode/Inspector/EntityListAdapter.cs

```diff
@@ -96,4 +97,12 @@ void DeleteAdapter(DCLBuilderInWorldEntity entityToEdit)
             if (entityToEdit.entityUniqueId == currentEntity.entityUniqueId)
                 Destroy(gameObject);
     }
+
+    private void ChangeEntityBoundsCheckerStatus(DCL.Models.DecentralandEntity entity, bool isInsideBoundaries)
+    {
+        if (currentEntity.rootEntity.entityId != entity.entityId)
+            return;
+
+        nameInputField_Text.color = isInsideBoundaries ? entityInsideOdBoundsColor : entityOutOfBoundsColor;
```

Comment:
```suggestion
        nameInputField_Text.color = isInsideBoundaries ? entityInsideOfBoundsColor : entityOutOfBoundsColor;
```
Typo in `entityInsideOdBoundsColor`.

Reply:
Ups, good catch! I will fix it in my other PR (https://github.com/decentraland/explorer/pull/2224) where I'm already modifying the `EntityListAdapter` prefab. Thanks!